modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -126,7 +125,7 @@ class packageSpec extends FlatSpec {
   it should "be in Running status after starting" in {
     val p = Process.eval(Task.now(start(seqId)))
     val qs = process(p)(qs1).take(1).runLast.unsafePerformSync.get._2
-    assert(qs.sequences(seqId).status === SequenceState.Running)
+    assert(qs.sequences(seqId).status == SequenceState.Running)
```

Comment:
Perhaps it's just that `Equal` is invariant.  What if you add an explicit type annotation `(SequenceState.Running: SequenceState)` or maybe add a constructor that produces a `SequenceState.Running` but typed as `SequenceState`?